api/types/src/transaction.rs

```diff
@@ -523,6 +555,16 @@ pub struct BlockMetadataTransaction {
     pub timestamp: U64,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
+pub struct SystemTransaction {
+    #[serde(flatten)]
+    #[oai(flatten)]
+    pub info: TransactionInfo,
+    pub events: Vec<Event>,
+    pub timestamp: U64,
+    pub txn_serialized: Vec<u8>,
```

Comment:
What is this txn_serialized field? Could you paste the output of an example transaction here? E.g. something like this: https://devnet.aptoslabs.com/v1/transactions/by_version/25100